kasa/iot/iotbulb.py

```diff
@@ -220,7 +220,7 @@ async def _initialize_modules(self):
             Module.IotAntitheft, Antitheft(self, "smartlife.iot.common.anti_theft")
```

Comment:
iotbulbs (at least my KL130) only support consumption:
```
        == Primary features ==
        State (state): False
        Current consumption (current_consumption): 0.0 W
        Voltage (voltage): None V
        Current (current): None A
        Brightness (brightness): 94 (range: 0-100)
        Color temperature (color_temperature): 2500 (range: 2500-9000)

        == Information ==
        Today's consumption (consumption_today): 0.0 kWh
        This month's consumption (consumption_this_month): 0.195 kWh
        Total consumption since reboot (consumption_total): None kWh
        Cloud connection (cloud_connection): False
```

Reply:
Yeah looking through the fixtures it seems fairly random so I'll update the `supported` for the iot devices to check for the whether the properties are in the module response. Probably in the `_post_update_hook` so it only happens once.

Reply:
Ok fixed with a check of `get_realtime` data